scripts/backCompatChecker.py

```diff
@@ -108,7 +110,13 @@ def findExtension():
             break
 
     if isArgumentMissing is True:
-        errorList.append('Argument missing for command ' + oldArgument.command + ' argument ' +  oldArgument.name)
+        allowedEntryFound = False
```

Comment:
nit: If block can be simplified to (do not need to reiterate the items since it is a dictionary) -

if not (allowedMissingArguments.get(oldArgument.command) and oldArgument.name in allowedMissingArguments.get(oldArgument.command)):
errorList.append('Argument missing for command ' + oldArgument.command + ' argument ' +  oldArgument.name)